src/bin/main.rs

```diff
@@ -9,12 +10,79 @@ use rusoto::error::*;
 use rusoto::sqs::*;
 use rusoto::s3::*;
 use rusoto::regions::*;
+use rusoto::dynamodb::DynamoDBHelper;
+use rusoto::dynamodb::CreateTableInput;
+use rusoto::dynamodb::AttributeDefinition;
+use rusoto::dynamodb::KeySchemaElement;
+use rusoto::dynamodb::DynamoDBError;
+use rusoto::dynamodb::PutItemInput;
+use rusoto::dynamodb::PutItemInputAttributeMap;
+use rusoto::dynamodb::AttributeValue;
+use std::thread;
 use time::*;
 use std::fs::File;
 use std::io::Write;
 use std::io::Read;
 
 fn main() {
+
+    let creds = DefaultAWSCredentialsProviderChain::new();
+    let region = Region::UsWest2;
+
+    let mut dynamodb = DynamoDBHelper::new(creds, &region);
+
+    match dynamo_list_tables_tests(&mut dynamodb) {
+        Ok(_) => {
+            println!("List tables OK");
+        }
+        Err(err) => {
+            println!("Error getting table list: {:#?}", err);
+        }
+    }
+
+    let table_name = &format!("test_table_{}", get_time().sec);
+
+    match dynamo_create_table_test(&mut dynamodb, &table_name) {
+        Ok(_) => {
+            println!("Issued create table command for {}", table_name);
+        }
+        Err(err) => {
+            println!("Error creating table {:#?}", err);
+        }
+    }
+
+    match dynamo_describe_wait_test(&mut dynamodb, &table_name) {
+        Ok(_) => {
+            println!("Table {} is now active", table_name);
+        }
+        Err(err) => {
+            println!("Error waiting for table to become active {:#?}", err);
+        }
+    }
+
+    match dynamo_put_item_test(&mut dynamodb, &table_name) {
+        Ok(_) => {
+            println!("Put item to {}", table_name);
+        }
+        Err(err) => {
+            println!("Error putting item to table {:#?}", err);
+        }
+    }
+
+
+    match dynamo_delete_table_test(&mut dynamodb, &table_name) {
+        Ok(_) => {
+            println!("Deleted table {}", table_name);
+        }
+        Err(err) => {
+            println!("Error deleting DynamoDB table {:#?}", err);
+        }
+    }
+    
+
+}
+
+fn old_main() {
```

Comment:
Yeah, our integration tests need to be completely refactored and split into modules at the very least.

I did some looking into this a while ago and got frustrated with the Rust community's attitude toward integration testing in general, which seems to be "Why aren't unit tests good enough for you?"  

Unless we want to put the time in to develop an integration testing framework, we might have to get creative.